openmc/data/library.py

```diff
@@ -4,6 +4,7 @@
 
 import h5py
 
+import openmc
```

Comment:
Is it worth importing just the config part of openmc
```suggestion
from openmc import config
```